.idea/libraries/.gitignore

```diff
@@ -1,2 +0,0 @@
-Cargo__rustfilt_.xml
```

Comment:
Empty folder, also there is no rustfilt installation or build in the main repo anymore